doc/sphinxext/custom_roles.py

```diff
@@ -4,8 +4,12 @@
 
 
 def rcparam_role(name, rawtext, text, lineno, inliner, options={}, content=[]):
-    param = rcParamsDefault.get(text)
-    rendered = nodes.Text(f'rcParams["{text}"] = {param!r}')
+    try:
```

Comment:
Is there a reason to use `try... except...` instead of `if text in rcParamsDefault`?